flake.nix

```diff
@@ -18,13 +18,23 @@
     prometheus-web.inputs.nixpkgs.follows = "nixpkgs";
     prometheus-web.inputs.ocaml-overlay.follows = "ocaml-overlays";
 
-    nix-npm-buildpackage.url = "github:serokell/nix-npm-buildpackage";
+    dream2nix.url = "github:nix-community/dream2nix";
   };
 
-  outputs = { self, nixpkgs, flake-utils, nix-npm-buildpackage, ocaml-overlays
-    , prometheus-web }:
-    with flake-utils.lib;
-    eachSystem defaultSystems (system:
+  outputs =
+    { self, nixpkgs, flake-utils, ocaml-overlays, prometheus-web, dream2nix }:
+    let
+      dream2nix-lib = dream2nix.lib2.init {
+        systems = [
+          "x86_64-linux"
+          "x86_64-darwin"
+          "aarch64-linux"
+          "aarch64-darwin"
+          "i686-linux"
+        ];
```

Comment:
```suggestion
        systems = flake-utils.lib.defaultSystems;
```

Reply:
Or we can define `supportedSystems` and omit `"i686-linux"`

Reply:
That would be ideal, as it seems to be part of the default systems in Flake utils

Reply:
`defaultSystems` are the systems that hydra tests on `nixpkgs` but we don't care about `i686-linux` here